pipeline/tag-release-pipeline.yaml

```diff
@@ -249,3 +249,42 @@ spec:
             resource: s2i-thoth
       runAfter:
         - tag-release-run
+    if not multiple environments
```

Comment:
seems like a comment is left , without comment syntax
will update it with proper working and syntax